aws_lambda_powertools/utilities/parser/parser.py

```diff
@@ -204,4 +202,4 @@ def handler(event: Order, context: LambdaContext):
             f"Error: {str(exc)}. Please ensure the Input model inherits from BaseModel,\n"
             "and your payload adheres to the specified Input model structure.\n"
             f"Model={model}",
-        ) from exc
+        )
```

Comment:
Please keep `from exc`, so we can propagate the original error to the customer.

Reply:
Thank you @leandrodamascena , reverted it back